content/en/docs/instrumentation/java/manual.md

```diff
@@ -704,18 +704,47 @@ instrument.
 
 ### Auto Configuration
 
-To configure the OpenTelemetry SDK based on the standard set of environment
-variables and system properties, you can use the
-`opentelemetry-sdk-extension-autoconfigure` module.
+Instead of manually creating the `OpenTelemetry` instance by using the SDK builders
```

Comment:
Remove the back ticks from OpenTelemetry

Reply:
The rationale for using the back ticks was about referring to the `OpenTelemetry` class not the OpenTelemetry "framework". This is at least what we do in our Strimzi documentation. But if you think we should not use them here, I can remove them. Let me know.

Reply:
now I get it, makes sense!